### DISCUSSION TITLE
chapter_appendix/terminology/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#904] [Announcements]
# chapter_appendix/terminology/

动画图解、一键运行的数据结构与算法教程

https://www.hello-algo.com/chapter_appendix/terminology/

**Comment by 51HzOuO** [comment]
第一脚

**Comment by Guohuixixi** [comment]
完结👍

**Reply by krahets** [reply]
撒花～

**Comment by hackerliuqiyuan** [comment]
再见

**Reply by krahets** [reply]
👋🎉

**Comment by Terran0629** [comment]
看完完结撒花咯！写的太棒了，完整的看一遍收获相应的知识点比刷多少题都不理解底层逻辑要好

**Reply by krahets** [reply]
撒花！

**Comment by beanljun** [comment]
完结撒花，收获很大，感谢作者

**Reply by krahets** [reply]
撒花～感谢你读完啦

**Comment by kaka59** [comment]
真的十分感谢作者，受益匪浅

**Comment by NaiveGhosttt** [comment]
完结撒花！！！

**Comment by Baldman-JYH** [comment]
学到很多，感谢作者分享！！
请问对于 第 16 章 附录 -> 16.3 术语表中，是否有必要对英文部分添加音标呢？

**Reply by krahets** [reply]
谢谢建议！感觉必要性不大，大多是基础单词，而且音标可以通过查字典获取

**Comment by zzzhty** [comment]
系统性过了一遍，收获很多！感谢作者分享~

**Comment by wkyfasttank** [comment]
请问怎么样才让GitHub停止给我发评论区邮件呢，我取消订阅你的Github仓库它还是给我发邮件

**Reply by krahets** [reply]
进入通知邮件里的链接，在对应页面点击“Unsubscribe”即可；或者尝试在仓库首页右上角将“Watch”选项改为“Ignore”。

GitHub 的消息通知与“Star”、“Fork”没有关联～

**Comment by summerBreeze03** [comment]
cache，简中叫它“缓存”，繁中叫“快取”，完全相反的意思却表示了同一个东西

**Reply by krahets** [reply]
哈哈哈，粗看还真是，“缓”可能代表“缓冲”的意思，不是“缓慢”

**Comment by mu-zi-lee** [comment]
花了几天过了一次这本书，之前看的是Algorithms这本。
我觉得这本书不管是用作开始学还是重新回顾都是很好的选择。
感谢k神 收获很大

**Reply by krahets** [reply]
谢谢鼓励！
